release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -883,6 +883,14 @@ Kubernetes 1.27 removed the following deprecated API, so you must migrate manife
 
 {product-title} {product-version} has removed support for the `OPENSHIFT_DEFAULT_REGISTRY` variable. This variable was primarily used to enable backwards compatibility of the internal image registry for earlier setups.
 
+[id="ocp-4-14-future-deprecation"]
+=== Notice of future deprecation
+
+[id="ocp-4-14-future-deprecation-sdn"]
+==== Future deprecation of the OpenShift SDN network plugin
+
+Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be an option for new installations, but will continue to be supported in clusters upgrading to 4.15 and 4.16 from clusters previously installed with the OpenShift SDN network plugin. In a future release, targeting no earlier than 4.17, the OpenShift SDN network plugin will be removed and and will no longer be supported.
```

Comment:
`clusters upgrading to 4.15 and 4.16` can be rewritten as `clusters upgrading till 4.16`. Not sure if i misunderstood something.

For e.g. a cu on 4.13 can upgrade till 4.16 on OpenshiftSDN and it will be supported

Reply:
Thanks, @anuragthehatter . I am asking Marc and team to review your comment.

Reply:
@anuragthehatter  No -- both versions must be specified.  Customers with an existing openshift-sdn cluster will be allowed to upgrade to 4.15 (e.g. from 4.14) or 4.16 (e.g. EUS customes that can upgrade directly from 4.14 to 4.16).   So, both 4.15 and 4.16 must be specified for that use case.